### PR TITLE
db optimization

### PR DESCRIPTION
**Pull request by LDouglasOT**
**Tasks Completed**
- created indices for the email on owners
- created indices for vet on visits

### Optimization of the following problematic queries
- #### SELECT COUNT(*) FROM visits where animal_id = 4;
  - Before optimization
animal_Id_BeforeOptimization

  - After optimization
animal_Id_afterOptimization

- #### SELECT * FROM visits where vet_id = 2;
  - Before optimization
Screenshot 2023-06-20 at 08 12 56
  - After optimization
Screenshot 2023-06-20 at 08 15 18

- SELECT * FROM owners where email = 'owner_18327@mail.com';
  - Before optimization
  Screenshot 2023-06-20 at 08 14 36

  - After optimization
emailOptimized

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -184,3 +184,4 @@ WHERE animals.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(*) DESC
 LIMIT 1;
+
```
Comment (gemmen29) [inline queries.sql:187]: - [ ] Kindly add the used insert statement to `data.sql` file as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=If%20you%20decide%20to%20commit%20any%20INSERT%20INTO%20queries%20%2D%20remember%20to%20add%20the%20m%20to%20the%20data.sql%20file).

File: schema.sql [hunk 2]
```diff
@@ -66,4 +66,6 @@ CREATE TABLE visits (
   FOREIGN KEY (vet_id) REFERENCES vets(id),
   FOREIGN KEY (animal_id) REFERENCES animals(id)
 );
+CREATE INDEX idx_email ON owners (email);
```
Comment (gemmen29) [inline schema.sql:70]: - [ ] Kindly add the missing index to your `schema.sql` file. The file should contain three indexes instead of two.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @LDouglasOT ,

Great job so far!  

May I highlight the following: 
- [x] Your  project looks awesome👏  ✔️ 
- [x] You implemented the previous TSE's requirements✔️ 





## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**